include/d/actor/d_a_ykgr.h

```diff
@@ -35,21 +39,59 @@ class daYkgr_c : public fopAc_ac_c {
     inline bool _execute();
     void hide() {}
     void setAimRate(float) {}
-    void setAlpha(unsigned char) {}
-    void set_mtx() {}
+    void setAlpha(unsigned char alpha) { m_alpha = alpha; }
+    void set_mtx() {
```

Comment:
I think usually these setmtx inlines would be defined in the cpp file because they're larger than most inlines. You can add the inline keyword to the declaration to make it still inline.
```suggestion
    inline void set_mtx();
```